includes/class-wc-order.php

```diff
@@ -105,7 +108,7 @@ public function payment_complete( $transaction_id = '' ) {
 			}
 			do_action( 'woocommerce_pre_payment_complete', $this->get_id() );
 
-			if ( ! empty( WC()->session ) ) {
+			if ( ! WC()->session ) {
```

Comment:
This check is the opposite of what it used to be. Did you mean `if ( WC()->session )`?